hack/run-kiali.sh

```diff
@@ -75,6 +75,7 @@ DEFAULT_API_PROXY_PORT="8001"
 DEFAULT_CLIENT_EXE="kubectl"
 DEFAULT_ENABLE_SERVER="true"
 DEFAULT_ISTIO_NAMESPACE="istio-system"
+DEFAULT_ISTIO_URL="http://127.0.0.1:15014/version"
```

Comment:
I believe you are missing the "D" in this variable ... "ISTIOD", not "ISTIO":
```suggestion
DEFAULT_ISTIOD_URL="http://127.0.0.1:15014/version"
```